server/block_processor.py

```diff
@@ -566,14 +567,14 @@ def backup_blocks(self, blocks):
         coin = self.coin
         for block in blocks:
             # Check and update self.tip
-            header = coin.block_header(block, self.height)
+            header, txs = coin.block_full(block, self.height)
             header_hash = coin.header_hash(header)
```

Comment:
Oh and that code was good enough to survive a forced testnet 3 block reorg via RPC